samples/meeting-av-filters/csharp/MeetingAVFilter/ClientApp/src/components/meeting-av-filter.css

```diff
@@ -0,0 +1,42 @@
+.horizontal {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+}
+
+.filter {
+    position: relative;
+    height: 42px;
+    border-radius: 2px;
+    border-width: 2px;
+    border-style: solid;
+    border-color: transparent;
+    margin: 2px;
+}
+
+.selected {
+    border-color: purple;
+    background: purple;
+}
+
+.thumbnail {
+    width: 74px;
+    height: 42px;
+    border-radius: 2px;
+}
+
+#filter-half > a {
+    width: 74px;
+    height: 42px;
+    background: url(http://localhost:3000/half.png);
```

Comment:
Why port number is hardcoded ?
If we are running the app on any other port, I think it will fail ?

Reply:
The code is configured in package.json to run on the same port.